### ISSUE TITLE
Why is it charging only at 21A?

### ISSUE THREAD

**Issue by SergioRius** [OpenEVSE/ESP32_WiFi_V4.x#552]
I've setup current shaper to 6300W (@28A) that is what my grid supports during the weekend.
The Max current is setup to 32A, and I have no limit configured on my M3LR.

Those are the sensor values been feed to openevse:
![image](https://user-images.githubusercontent.com/1693727/221402273-8bba91aa-8384-4a9d-9c5e-f5ba69e32bef.png)

And this is the status during the charge. It tops at @20A and doesn't move from there, up or down:
![image](https://user-images.githubusercontent.com/1693727/221402337-f62d963f-befc-488c-b1c5-c37c3ac295ce.png)

Nothing changes. Asides from that grid import, my panels are producing around 2900w ATM, that are going into my home battery. So I can't understand this limit.
Could be something I've done?

PS: If I set the current shaper to 9000w charge raises to 21.12A and nothing more. (dropping to 20.90 at times)
If I disable current shaper, the pilot value reads 28A (Max configured), but the charge still doesn't raise from 21A.

Thanks for your help.

EDIT: If I stop the charge and restart the wifi module it goes charging up to 27.94A.
Then if I move max charge to 30, it applies to pilot correctly.
Then I go again activating the shaper at 6300w and current doesn't drop.
**The current shaper is setup to 4600w during the labor week. Could be that changing the shaper rate needs a reboot?**

**Comment by SergioRius** [comment]
Although I managed to charge at full speed, after a while it drops to 24A.
The temperature is reading 51ºC, it the Openevse throttling on temperature?

Restarting the charge makes it max the charge rate, but after a little while it drops again.

**Comment by SergioRius** [comment]
Ok, it seems the car.
My grid seems to be slightly weak, so voltage drops. As soon as voltage drops below 220 the car throttles.
I've adjusted the PV system to give priority to production over grid (it was configured for dawn) and as import lowered, voltage raised and thus charging rate.

The only thing I still don't understand is why the pilot goes to 6A when I select 32A on Openevse. I tried changing service level from auto to 2 and now I also can't select 31A even after revert to auto. I can only select up to 30A.
